applications/tari_dan_wallet_cli/src/command/transaction.rs

```diff
@@ -351,6 +366,44 @@ pub async fn handle_claim_burn(
     submit_transaction(instructions, common, client).await
 }
 
+pub async fn handle_send(
+    args: SendArgs,
+    client: &mut WalletDaemonClient,
+) -> Result<TransactionSubmitResponse, anyhow::Error> {
+    let SendArgs {
+        source_account_name,
+        amount,
+        resource_address,
+        dest_address,
+        common,
+    } = args;
+
+    let source_address = client.get_by_name(source_account_name).await?;
+    let source_component_address = if let SubstateAddress::Component(ca) = source_address.account_address {
+        ca
+    } else {
+        return Err(anyhow!("Invalid component address for source address"));
+    };
+
+    let instructions = vec![
+        Instruction::CallMethod {
+            component_address: source_component_address,
+            method: String::from("withdraw"),
+            args: args![resource_address, amount],
```

Comment:
The type for the second arg is Amount(i64). I dont mind that the cli arg is u128 (should definitely be unsigned) but the correct type needs to be passed in args.

I think it'll be easier if you make it a u32 since that can be infallibly converted.
```suggestion
            args: args![resource_address, Amount::from(amount)], // amount is u32
```